content/docs/install/helm-install/index.md

```diff
@@ -72,14 +133,65 @@ helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=
        ingress.tls=<YOUR_TLS>
   ```
 
-## Install Keptn execution-plane:
+## Confirm Installation
+
+After you issue the **helm install** command,
+it takes a couple of minutes for the installation to finish.
+Use the following command to watch the progress:
+
+```
+kubectl -n keptn get pods
+```
+
+Wait for all the pods in the Keptn namespace to show `Running` under the `STATUS` column
+before proceeding.
+
+Use the following command to view all the services that are installed in the Keptn namespace:
+
+```
+kubectl -n keptn get services
+```
+
+## Access the Keptn Bridge
+
+The [Keptn Bridge](../../0.18.x/bridge) is the graphical user interface
+you can use to manage and view Keptn projects running in your installation.
+To access it:
+
+1. Expose the API Gateway NGINX.  This can be done in two ways:
+
+   * Set the `apiGatewayNginx` parameter during installation
+   * Issue the following command after installation:
+     ```
+     kubectl -n keptn port-forward svc/api-gateway-nginx 8080:80
+     ```
+2. Open a browser window to `localhost:8080`
+
+3. Log into the Keptn Bridge.
+   The following commands give you the username and randomly-generated password
+   to use if your site uses [Basic Authentication](../../0.18.x/bridge/basic_authentication):
+
+   ```
+   kubectl -n keptn get secret bridge-credentials -o jsonpath-{.data.BASIC_AUTH_USERNAME} | base64 -d echo
+   kubectl -n keptn get secret bridge-credentials -o jsonpath-{.data.BASIC_AUTH_PASSWORD} | base64 -d echo
+   ```
+   You can also use [OpenID Authentication](../../0.18.x/bridge/oauth) to access the Keptn Bridge.
+
+## Install Execution Plane
+
+To install the Execution Plane in the same namespace as the Control Plane,
+install a microservice.
+For example, the following two commands install the Jmeter and Helm-service microservices:
 
 ```
 helm install jmeter-service https://github.com/keptn/keptn/releases/download/0.18.1/jmeter-service-0.18.1.tgz -n keptn --create-namespace --wait
 
 helm install helm-service https://github.com/keptn/keptn/releases/download/0.18.1/helm-service-0.18.1.tgz -n keptn --create-namespace --wait
```

Comment:
Just a heads-up: The 2 URLs here will change with Keptn 0.19 since helm-service and jmeter-service were moved to the keptn-contrib GH organization now.

Reply:
https://github.com/keptn/keptn.github.io/issues/1395